charts/pypi/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for pypi.
+# This is a YAML-formatted file.
+
+replicaCount: 1
+
+image:
+  repository: codekoala/pypi
+  tag: 1.2.1
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  # annotations:
+  #   kubernetes.io/ingress.class: nginx
+  #   kubernetes.io/tls-acme: "true"
+  #   nginx.ingress.kubernetes.io/auth-type: basic
+  #   # name of the secret that contains the user/password definitions
+  #   nginx.ingress.kubernetes.io/auth-secret: basic-auth-htpasswd
+  #   nginx.ingress.kubernetes.io/auth-realm: "Authentication Required"
+  # path: /
+  # hosts:
+  #   - myhost.com
+  # tls:
+  #   - secretName: myhost.com-tls
+  #     hosts:
+  #       - myhost.com
+
+ingressUpload:
+  enabled: false
+  # annotations:
+  #   kubernetes.io/ingress.class: nginx
+  #   kubernetes.io/tls-acme: "true"
+  #   nginx.ingress.kubernetes.io/auth-type: basic
+  #   # name of the secret that contains the user/password definitions
+  #   nginx.ingress.kubernetes.io/auth-secret: basic-auth-upload-htpasswd
+  #   # Authenticate to backend which has basic auth
+  #   nginx.ingress.kubernetes.io/configuration-snippet: |
+  #     proxy_set_header Authorization $http_authorization;
+  #     proxy_pass_header  Authorization;
+  #   nginx.ingress.kubernetes.io/proxy-body-size: "50m"
+  # path: /
+  # hosts:
+  #   - upload.myhost.com
+  # tls:
+  #   - secretName: upload.myhost.com-tls
+  #     hosts:
+  #       - upload.myhost.com
+
+basicAuth:
+  # Authenticate for uploading, use existing secret
+  secretName: "basic-auth-upload-htpasswd"
```

Comment:
How can it pass the test?

Reply:
It has been reworked : https://github.com/wiremind/wiremind-helm-charts/commit/30b3a5ff4c6414ccb3361aee7cdabf3d1a0a9a06#diff-3501b229ecff8a86900788c6eaf849bd7c8c88df561e718e83b766f5189b04d8